packages/Search/src/DTK_DistributedSearchTree_decl.hpp

```diff
@@ -32,6 +32,33 @@ class DistributedSearchTree
         Teuchos::RCP<Teuchos::Comm<int> const> comm,
         Kokkos::View<Box const *, DeviceType> bounding_boxes );
 
+    /** \brief Finds object meeting the passed predicates (e.g. nearest to some
```

Comment:
Bruno had the same comment.  Boost talked about "meeting predicates" but I am fine changing it if y'all don't like that.